Week3/HW3/3_moreJavascript/14_langTaught.js

```diff
@@ -0,0 +1,8 @@
+let moduleTeachers = {};
+moduleTeachers = [
+    { name: 'Philipp Beau', language: 'HTML'},
+    { name: 'Rob van Kruijsdijk', language: 'CSS'},
+    { name: 'Unmesh Joshi', language: 'CLI'},
+    { name: 'Yash Kapila', language: 'Java Script 1'},
+    {name: 'Wilgert Velinga', language: 'Java Script 2'}
+  ];
```

Comment:
Oh I see you corrected it here 😄 

Please see my comment on previous exercise and see if you could combine the `languages` and `name` in a single data structure as explained.

Reply:
Yes, I changed and corrected it, too.